core/src/main/java/org/bitcoinj/core/TransactionConfidence.java

```diff
@@ -415,6 +417,16 @@ public synchronized String toString() {
                 builder.append("  InstantSendLock: Unknown status");
                 break;
         }
+
+        InstantSendLock islock = getInstantSendlock();
+        if (islock != null) {
+            BLSLazySignature isLockSig = islock.getSignature();
+            if (isLockSig != null) {
+                builder.append(" [sig=" + isLockSig +"]");
+            } else {
+                builder.append(" [sig=null]");
+            }
```

Comment:
This "if" doesn't seem to be necessary, when isLockSig is null it should be replaced with "null" string by default.

Reply:
That sounds right, but I was getting some `NullPointerException`s earlier so I did it like this.